### DISCUSSION TITLE
google play app installation without linux

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#541] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there any way to install apk files without linux or dev mode. because I have a lot of apps that could be useful if I could install them

**Comment by xHydra1337** [comment, marked as answer]
No it really isnt, unless you can find a way to run ADB debugging without the terminal provided by default, then it would be super hard. Nothing is impossible, but you just need to find a way to run linux on your chromebook then run ADB debugging. Your best bet is honestly to just unenroll on sh1mmer.
